enterprise-ws-spec/src/main/asciidoc/Overview.adoc

```diff
@@ -40,96 +40,98 @@ with the following characteristics:
 the Port are transparent to the client. The client only needs to make
-method calls on the Service Endpoint Interface, as defined by JAX-RPC or
-JAX-WS, (i.e. PortType) to access the service. See Chapter
+method calls on the Service Endpoint Interface, as defined by
+Jakarta XML Web Services, (i.e. PortType) to access the service. See Chapter
 link:#anchor-6[4] for more details.
 
 image:2.png[image,width=277,height=135]
 
 * [#anchor-5]##Figure 2 Client view
 
-=== Web Services for _*Java EE*_ Overview
+=== Web Services for _*Jakarta EE*_ Overview
 
-The Web Services for Java EE specification defines the required
+The Web Services for Jakarta EE specification defines the required
 architectural relationships as shown in link:#anchor-7[Figure 3]. This
 is a logical relationship and does not impose any requirements on a
 container provider for structuring containers and processes. The
-additions to the Java EE platform include a port component that depends
+additions to the Jakarta EE platform include a port component that depends
 on container functionality provided by the web and EJB containers, and
 the SOAP/HTTP transport.
 
+//TODO: image 3.png mentions J2EE - should be replaced with Jakarta EE?
+
 image:3.png[image,width=371,height=280]
 
-* [#anchor-7]##Figure 3 Java EE architecture diagram
+* [#anchor-7]##Figure 3 Jakarta EE architecture diagram
 
-Web Services for Java EE requires that a Port be referencable from the
+Web Services for Jakarta EE requires that a Port be referencable from the
 client, web, and EJB containers. This specification does not require
 that a Port be accessible from the applet container.
 
-This specification adds additional artifacts to those defined by JAX-RPC
-or JAX-WS that may be used to implement Web services, a role based
-development methodology, portable packaging and Java EE container
+This specification adds additional artifacts to those defined by
+Jakarta XML Web Services that may be used to implement Web services, a role
+based development methodology, portable packaging and Jakarta EE container
 services to the Web services architecture. These are described in later
 sections.
 
 ==== Web Service Components
 
 This specification defines two means for implementing a Web service,
-which runs in a Java EE environment, but does not restrict Web service
+which runs in a Jakarta EE environment, but does not restrict Web service
 implementations to just those means. The first is a container based
-extension of the JAX-RPC or JAX-WS programming model which defines a Web
+extension of the Jakarta XML Web Services programming model which defines a Web
 service as a Java class running in the web container. The second uses a
 constrained implementation of a stateless session EJB or singleton
-session EJB(only for JAX-WS services) in the EJB container. Other
-service implementations are possible, but are not defined by this
+session EJB(only for Jakarta XML Web Services services) in the EJB container.
```

Comment:
Since JAX-RPC is out of the game, '(only for Jakarta XML Web Services services)' can be removed. If you don't agree, there should be a space before '(' at least. Thx